tests/test_clip.py

```diff
@@ -22,37 +23,42 @@ def test_clip_crs(self):
             self.assertEqual(src_clip.crs, src.crs)
 
     def test_clip_data(self):
-        with gw.open(l8_224077_20200518_B2, chunks=64) as src:
-            src_crs = src.crs
-            src_clip = gw.clip_by_polygon(src, l8_224078_20200518_polygons)
-            src_clip_mask = gw.clip_by_polygon(
-                src, l8_224078_20200518_polygons, mask_data=True
-            )
-        df = gpd.read_file(l8_224078_20200518_polygons)
-        df = df.to_crs(src_crs)
-        with gw.config.update(
-            ref_bounds=df.total_bounds.tolist(), ref_tar=l8_224077_20200518_B2
-        ):
-            with gw.open(l8_224077_20200518_B2, chunks=64) as ref:
-                self.assertTrue(
-                    np.allclose(ref.data.compute(), src_clip.data.compute())
+        with warnings.catch_warnings():
+            warnings.simplefilter('ignore', DeprecationWarning)

```

Comment:
dont we want deprecation warnings to show up here?  Really only you and I will see them. Might help us stay on top of future issues.

Reply:
I reverted the warning catches.